### ISSUE TITLE
HRTF tables: Could not open/Incompatible Format

### ISSUE THREAD

**Issue by Hiradur** [kcat/openal-soft#17]
* tested with build no. 1486 from http://ci.newdawnsoftware.com/view/OpenAL/job/OpenAL-win32/ on Windows 7 x64
* For current master both default HRTF and self made (makehrtf) HRTF tables don't work. In 1.16.0 they work fine.

Logs with Loglevel=3:
nightly build:

```
AL lib: (II) UpdateDeviceParams: Post-reset: Stereo, Float, 44100hz, 896 update size x4
AL lib: (II) GetConfigValue: Found stereo-mode = "headphones"
AL lib: (II) GetConfigValue: Key hrtf-mode not found
AL lib: (II) GetConfigValue: Found hrtf_tables = "hiradur-%r.mhr"
AL lib: (II) LoadHrtf: Loading hiradur-44100.mhr...
AL lib: (II) OpenDataFile: Opened C:\Users\Hiradur\AppData\Roaming\openal\hrtf\hiradur-44100.mhr
AL lib: (EE) LoadHrtf: Could not open hiradur-44100.mhr
AL lib: (EE) GetHrtf: Incompatible format: Stereo 44100hz
AL lib: (II) UpdateDeviceParams: HRTF disabled
```

1.16.0:
```
AL lib: (II) UpdateDeviceParams: Pre-reset: Stereo, Float, 44100hz, 1024 update size x4
AL lib: (II) GetConfigValue: Found hrtf = "true"
AL lib: (II) GetConfigValue: Found hrtf_tables = "hiradur-%r.mhr"
AL lib: (II) LoadHrtf: Loading hiradur-44100.mhr...
AL lib: (WW) OpenDataFile: Could not open hiradur-44100.mhr
AL lib: (II) OpenDataFile: Opened C:\Users\Hiradur\AppData\Roaming\openal\hrtf\hiradur-44100.mhr
AL lib: (II) LoadHrtf: Detected data set format v1
AL lib: (II) LoadHrtf: Loaded HRTF support for format: Stereo 44100hz
AL lib: (II) ALCmmdevProxy_messageHandler: Got message 1025
AL lib: (II) UpdateDeviceParams: Post-reset: Stereo, Float, 44100hz, 896 update size x4
AL lib: (II) GetConfigValue: Key layout_stereo not found
AL lib: (II) GetConfigValue: Key layout not found
AL lib: (II) UpdateDeviceParams: HRTF enabled
```

**Comment by kcat** [comment]
Looks like it was a bad return value. Should be fixed in commit 45703455342ced95bbbc9ee20b91716531e63b48, if you could check?

**Comment by Hiradur** [comment]
https://github.com/kcat/openal-soft/commit/45703455342ced95bbbc9ee20b91716531e63b48 fixed it, thanks!
